server/datastore/mysql/migrations/tables/20230927111208_AddIndexToHostScriptResults.go

```diff
@@ -0,0 +1,28 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20230927111208, Down_20230927111208)
+}
+
+func Up_20230927111208(tx *sql.Tx) error {
```

Comment:
Just a heads-up to make sure the migrations get added correctly when merging to `main` (I think it basically means added as latest timestamp to the list of existing ones). Since this is a feature branch where the migration was created a while ago, while `main` kept on getting new commits, I'm not sure if the CI check would catch any issues with that, so something to keep an eye on.

Reply:
great call out, I also don't trust the CI check as it's pretty simplistic.

seems like the latest migration in `main` is `20230918221115_MoveDiskEncryptionSetting_test.go` so we're good now, but we should keep an eye in case something is merged in between.